aspnet/web-api/overview/odata-support-in-aspnet-web-api/odata-v3/working-with-entity-relations.md

```diff
@@ -1,8 +1,8 @@
 ---
 uid: web-api/overview/odata-support-in-aspnet-web-api/odata-v3/working-with-entity-relations
 title: "Supporting Entity Relations in OData v3 with Web API 2 | Microsoft Docs"
-author: MikeWasson
-description: "Most data sets define relations between entities: Customers have orders; books have authors; products have suppliers. Using OData, clients can navigate over..."
+author: rick-anderson
+description: "The majority of data sets define relations between entities: Customers have orders; books have authors; products have suppliers. Using OData, clients can navigate over..."
```

Comment:
Description is too long. Can you rewrite this? Looks like this description was originally unfinished.